src/containers/teamPage/index.tsx

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, Collapse, List, Typography } from 'antd';
+import { Routes } from '../../App';
+import { GoalProps, MemberProps, TeamProps, TeamRole } from './ducks/types';
+import GoalInfo from '../../components/goalInfo';
+import PageHeader from '../../components/pageHeader';
+import TeamMember from '../../components/teamMember';
+import { LinkButton } from '../../components/LinkButton';
+import { getDateString } from '../../utils/stringFormat';
+import {
+  BLACK,
+  DARK_GREEN,
+  LIGHT_GREEN,
+  MID_GREEN,
+  WHITE,
+} from '../../utils/colors';
+
+const { Paragraph } = Typography;
+const { Panel } = Collapse;
+
+const TeamContainer = styled.div`
+  padding: 70px 134px;
+`;
+
+const StyledLinkButton = styled(LinkButton)`
+  width: 190px;
+  height: 45px;
+  margin-bottom: 20px;
+  border-color: ${MID_GREEN};
+  font-size: 18px;
+  color: ${MID_GREEN};
+`;
+
+const TeamHeaderContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 80px;
+`;
+
+const PageHeaderContainer = styled.div`
+  max-width: 600px;
+`;
+
+const JoinButton = styled(Button)`
+  width: 125px;
+  height: 45px;
+  color: ${WHITE};
+  background-color: ${MID_GREEN};
+  font-size: 18px;
+`;
+
+const CenterDiv = styled.div`
+  margin-top: -50px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const MemberContainer = styled.div`
+  width: 45%;
+`;
+
+const GoalContainer = styled.div`
+  width: 55%;
+`;
+
+const SectionHeader = styled(Paragraph)`
+  font-size: 25px;
+  font-weight: bold;
+  line-height: 1;
+  color: ${DARK_GREEN};
+`;
+
+const StyledMemberList = styled(List)`
+  border-bottom: 0px;
+`;
+
+const LeaderboardCollapse = styled(Collapse)`
+  color: ${BLACK};
+  font-size: 15px;
+`;
+
+const BlackText = styled(Paragraph)`
+  display: inline-block;
+  color: ${BLACK};
+  font-size: 15px;
+  line-height: 0.5;
+`;
+
+const StyledPanel = styled(Panel)`
+  min-height: 40px;
+  margin-bottom: 10px;
+  padding-left: 10px;
+  background-color: ${LIGHT_GREEN}90;
+  border: 0px;
+`;
+
+const Line = styled.div`
+  height: 1px;
+  width: 104%;
+  margin: 0px -10px 10px -20px;
+  display: block;
+  background-color: ${WHITE};
+`;
+
+// dummy data
```

Comment:
📌 Where is the work to connect this to the API ticketed? I can pair on implementing a Redux store for this data.

Reply:
[Monday](https://code4community-team.monday.com/boards/925293934/views/18904377/pulses/926123296)